lesson_06/expression/src/expression_calculator.ts

```diff
@@ -2,10 +2,36 @@ export class ExpressionCalculator {
   /** Returns the calculation of ((a + b) * c) / d^e */
   calculate(a: number, b: number, c: number, d: number, e: number): number {
     // Implement your code here to return the correct value.
-    return 0;
+    const Sum = this.add(a, b)
```

Comment:
Your variable names should be `lowerCamelCase`, not `ProperCase`.

Reply:
corrected